script.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
did you mean to commit this to the repo?

Reply:
Another good catch. I swear I was being careful